### ISSUE TITLE
UI: javascript error - no flags or segments

### ISSUE THREAD

**Issue by giddel** [markphelps/flipt#394]
**Describe the bug**
Version: v1.2.0
After updating Flipt from 0.18.0 to 1.2.0 no Flags or Segments are visible in UI:
![No-Flags-or-Segments](https://user-images.githubusercontent.com/10463018/110338339-2eab7300-8027-11eb-8f3c-d7683e117099.png)

There are  javascript errors in browser console. 
**Screenshots**
Error on Flags:
![Js-Error-Flags](https://user-images.githubusercontent.com/10463018/110336874-95c82800-8025-11eb-8aec-1195c50e1d2e.png)
Error on Segments:
![JS-Error-Segments](https://user-images.githubusercontent.com/10463018/110336887-98c31880-8025-11eb-9a9d-eeb80411a12a.png)

But the API is working fine all,  validation still works.
After switch back to version v1.1.0 Flags and Segments are visible again and the javascript erros disappear.

**Comment by giddel** [comment]
Oh sorry ... seems to be in work here: https://github.com/markphelps/flipt/pull/388#issuecomment-782121157

**Comment by markphelps** [comment]
@giddel does `0.0.0.0:8080` work instead of `127.0.0.1:8080`?

**Comment by giddel** [comment]
Not really. 
I'm using Flipt with k8s.
My default.yaml contains:
```
    db:  <database settings>
    log:
      level: DEBUG
    server:
      host: 127.0.0.1
```
The pod comes not healthy (8080 is the Flipt http port)
```
  Warning  Unhealthy  3m8s (x4 over 3m38s)   kubelet            Readiness probe failed: Get http://10.8.24.172:8080/: dial tcp 10.8.24.172:8080: connect: connection refused
  Warning  Unhealthy  3m8s (x4 over 3m38s)   kubelet            Liveness probe failed: Get http://10.8.24.172:8080/: dial tcp 10.8.24.172:8080: connect: connection refused
```
And Flipt itself says:
```
 _____ _ _       _
|  ___| (_)_ __ | |_
| |_  | | | '_ \| __|
|  _| | | | |_) | |_
|_|   |_|_| .__/ \__|
          |_|

Version: 1.2.0
Commit: 049e6ab29d47b04992f3338de5da43993b4bfb65
Build Date: 2021-03-08T19:36:24Z
Go Version: go1.15.5

time="2021-03-08T19:36:26Z" level=debug msg="checking for updates..."
time="2021-03-08T19:36:28Z" level=debug msg="current version: 1.2.0; latest version: 1.2.0"
You are currently running the latest version of Flipt [1.2.0]!
time="2021-03-08T19:36:28Z" level=debug msg="migrations up to date"
time="2021-03-08T19:36:28Z" level=debug msg="starting grpc server" server=grpc store=mysql
time="2021-03-08T19:36:29Z" level=debug msg="starting http server" server=http

API: http://127.0.0.1:8080/api/v1
UI: http://127.0.0.1:8080

time="2021-03-08T19:36:51Z" level=info msg="shutting down..."
time="2021-03-08T19:36:51Z" level=info msg="server shutdown gracefully" server=http
```
The shutdown is caused by k8s imho, because the failed readiness probe ... I have to try some more about this failing probe ...

**Comment by giddel** [comment]
Flipt v1.1.0 came up only(!) without the server.host entry in `default.yaml`.

**Comment by markphelps** [comment]
Update: I have a fix for this in #396. Once the tests run/pass I'll release v1.2.1
